packages/release-manifest/src/manifest.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fetch from 'node-fetch';
+
+const VERSIONS_DOMAIN = 'https://versions.backstage.io';
+
+/**
+ * Contains mapping between Backstage release and package versions.
+ * @public
+ */
+export type ReleaseManifest = {
```

Comment:
This should contain `releaseVersion` now as well right?

Reply:
It should but it's not used yet. I'll add it 👍